src/RTTI/RTTI_TypeIDs.hpp

```diff
@@ -5,12 +5,43 @@ namespace REGoth
   // See BsCorePrerequisites.h, TypeID_Core
   enum TypeID_REGoth
   {
-    TID_REGOTH_VisualCharacter         = 60000,
-    TID_REGOTH_Character               = 60001,
-    TID_REGOTH_ScriptBackedBy          = 60002,
-    TID_REGOTH_CharacterKeyboardInput  = 60003,
-    TID_REGOTH_CharacterAI             = 60004,
-    TID_REGOTH_VisualSkeletalAnimation = 60005,
-    TID_REGOTH_VisualInteractiveObject = 60006,
+    TID_REGOTH_VisualCharacter              = 600000,
+    TID_REGOTH_Character                    = 600001,
+    TID_REGOTH_ScriptBackedBy               = 600002,
+    TID_REGOTH_CharacterKeyboardInput       = 600003,
+    TID_REGOTH_CharacterAI                  = 600004,
+    TID_REGOTH_VisualSkeletalAnimation      = 600005,
+    TID_REGOTH_VisualInteractiveObject      = 600006,
+    TID_REGOTH_ScriptObjectStorage          = 600007,
+    TID_REGOTH_ScriptObject                 = 600008,
+    TID_REGOTH_Waypoint                     = 600009,
+    TID_REGOTH_Waynet                       = 600010,
+    TID_REGOTH_VisualStaticMesh             = 600011,
+    TID_REGOTH_NodeVisuals                  = 600012,
+    TID_REGOTH_Item                         = 600013,
+    TID_REGOTH_VisualMorphMesh              = 600014,
+    TID_REGOTH_GameWorld                    = 600015,
+    TID_REGOTH_NeedsGameWorld               = 600016,
+    TID_REGOTH_ScriptVM                     = 600017,
+    TID_REGOTH_DaedalusVM                   = 600018,
+    TID_REGOTH_ScriptVMForGameWorld         = 600019,
+    TID_REGOTH_DaedalusVMForGameWorld       = 600020,
+    TID_REGOTH_ScriptInts                   = 600021,
+    TID_REGOTH_ScriptFloats                 = 600022,
+    TID_REGOTH_ScriptStrings                = 600023,
+    TID_REGOTH_ScriptSymbolStorage          = 600024,
+    TID_REGOTH_ScriptSymbolBase             = 600025,
+    TID_REGOTH_ScriptSymbolInt              = 600026,
+    TID_REGOTH_ScriptSymbolFloat            = 600027,
+    TID_REGOTH_ScriptSymbolString           = 600028,
+    TID_REGOTH_ScriptSymbolClass            = 600029,
+    TID_REGOTH_ScriptSymbolScriptFunction   = 600030,
+    TID_REGOTH_ScriptSymbolExternalFunction = 600031,
+    TID_REGOTH_ScriptSymbolPrototype        = 600032,
+    TID_REGOTH_ScriptSymbolInstance         = 600033,
+    TID_REGOTH_ScriptSymbolUnsupported      = 600034,
+    TID_REGOTH_ScriptClassTemplates         = 600035,
+    TID_REGOTH_ScriptObjectMapping          = 600036,
+	TID_REGOTH_GameClock					= 600037,
```

Comment:
Used tabs instead of spaces?

Reply:
Ohh. I have VSCodium set up correctly, but this change I did with BeyondCompare.
Need to change the settings there, too.